csharp/global.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "sdk": {
+    "version": "1.0.0"
```

Comment:
Suppose someone clones this repository under a directory which already has a global.json file in, pointing at the beta02 SDK - then if we don't have a global.json, it will try to use the beta02 SDK. This prevents that problem.